docs/production-deployment/cloud/service-health.mdx

```diff
@@ -0,0 +1,113 @@
+---
+id: service-health
+title: Service health
+sidebar_label: Service health
+description: Use Temporal Cloud metrics to monitor your production deployment Service health.
+toc_max_heading_level: 4
+tags:
+  - temporal-cloud
+  - metrics
+  - observability
+  - health
+---
+
+Temporal Cloud metrics help you to monitor the Service health of your production deployments.
+
+## Monitor availability issues
+
+When you see a sudden drop in Worker resource utilization, verify whether Temporal Cloud's API is showing increased latency and error rates.
+
+### Reference Metrics
+
+- [temporal\_cloud\_v0\_service\_latency\_bucket](/production-deployment/cloud/metrics/reference#temporal_cloud_v0_service_latency_bucket)
+
+:::note
+
+The Service latency metric only supports a subset of the operations we consider mission critical, namely SignalWithStartWorkflowExecution, SignalWorkflowExecution, and StartWorkflowExecution.
+
+:::
+
+### Prometheus Query
+
+P99 service lag (histogram):
+
+```
+histogram_quantile(0.99, sum(rate(temporal_cloud_v0_service_latency_bucket[$__rate_interval])) by (temporal_namespace, operation, le))
+```
+
+## Monitor Temporal Service errors
+
+Check for front-end Temporal Service API errors.
+Note that Service API errors are not equivalent to guarantees mentioned in the Temporal Cloud SLA.
+
+### Reference Metrics
+
+- [temporal\_cloud\_v0\_frontend\_service\_error\_count](/production-deployment/cloud/metrics/reference#temporal_cloud_v0_frontend_service_error_count)
+- [temporal\_cloud\_v0\_frontend\_service\_request\_count](/production-deployment/cloud/metrics/reference#temporal_cloud_v0_frontend_service_request_count)
+
+
+### Prometheus Query
+
+Measure your daily average errors over 10-minute windows:
+
+```
+avg_over_time((
+    (
+
+        (
+            sum(increase(temporal_cloud_v0_frontend_service_request_count{temporal_namespace=~"$namespace", operation=~"StartWorkflowExecution|SignalWorkflowExecution|SignalWithStartWorkflowExecution|RequestCancelWorkflowExecution|TerminateWorkflowExecution"}[10m]))
+            -
+            sum(increase(temporal_cloud_v0_frontend_service_error_count{temporal_namespace=~"$namespace", operation=~"StartWorkflowExecution|SignalWorkflowExecution|SignalWithStartWorkflowExecution|RequestCancelWorkflowExecution|TerminateWorkflowExecution"}[10m]))
+        )
+        /
+        sum(increase(temporal_cloud_v0_frontend_service_request_count{temporal_namespace=~"$namespace", operation=~"StartWorkflowExecution|SignalWorkflowExecution|SignalWithStartWorkflowExecution|RequestCancelWorkflowExecution|TerminateWorkflowExecution"}[10m]))
+    )
+    
+    or vector(1)
+
+    )[1d:10m])
+```
+
+## Monitor multi-region Namespace replication lag
```

Comment:
Just pointing out that this how-to guidance is in service health, but there is a section on explaining replication lag in more detail on a separate page...

https://github.com/temporalio/documentation/pull/2884/files#diff-14d414288d536e46d722cf8a38cc65c29a04b0ca340c945e2b33b93809f83ad0R416

Reply:
Your previous point is important. I'd like to ship and then talk about IA and your vision for where each kind of content lives, and shuffle things around now that we're past the deadline.